include/seccomp.h.in

```diff
@@ -816,6 +816,19 @@ int seccomp_export_pfc(const scmp_filter_ctx ctx, int fd);
  */
 int seccomp_export_bpf(const scmp_filter_ctx ctx, int fd);
 
+/**
+ * Generate seccomp Berkley Packet Filter (BPF) code and export it to a buffer
+ * @param ctx the filter context
+ * @param buf the destination buffer
+ * @param len the input buf len, and the output buf len
```

Comment:
When describing `len` I might suggest a tweak to the language.  Instead of:
> the input buf len, and the output buf len

how about the following?
> on input the length of the buffer, on output the number of bytes written to the buffer